### DISCUSSION TITLE
License conflict

### DISCUSSION THREAD

**Discussion by chrisbasmajian** [adobe/react-spectrum#6028] [Q&A] [answered]
When I install react-spectrum, which is licensed under [Apache-2.0 license](https://github.com/adobe/react-spectrum/discussions/new?category=q-a#Apache-2.0-1-ov-file), it also installs react-spectrum-ui, which is licensed under [CC-BY-ND-4.0](https://creativecommons.org/licenses/by-nd/4.0/deed.en). These seems to be in conflict, I can't use CC-BY-ND-4.0.

**Comment by dannify** [comment]
Hey @chrisbasmajian 
What issues are you having with the difference in licensing? Why can you not use CC-BY-ND-4.0 specifically?

**Reply by chrisbasmajian** [reply]
My organization does not allow use of the CC-BY-ND-4.0, the Apache 2.0 license is more permissive.

**Reply by domdomegg** [reply]
I think an issue is that it's unclear how the 'no-derivatives' part of the CC license should be interepreted for software. For example, is an app that uses the react spectrum library a derivative (plausibly yes, in the same way that say using an image in a set of presentation slides creates a derivative: it's included in some other intellectual work). Given this legal uncertainty, it'd be better to relicense the underlying software as either CC-BY-4.0, or ideally Apache 2 (so it's the same as the rest of react-spectrum).

For what it's worth, Creative Commons recommend _against_ using their licenses for software: https://creativecommons.org/faq/#can-i-apply-a-creative-commons-license-to-software

**Reply by snowystinger** [reply, marked as answer]
Great news, we're updating the license https://github.com/adobe/react-spectrum/pull/6735
